receipt_dynamo_stream/receipt_dynamo_stream/exceptions.py

```
@@ -0,0 +1,64 @@
+            f"'{queue_name}' queue",
+            queue_name=queue_name,
+        )
+
+
+__all__ = [
+    "ReceiptDynamoStreamError",
+    "QueuePublishError",
+    "QueueConfigurationError",
+    "QueueServiceError",
+    "QueueBatchFailureError",
+]
```

Ruff (0.16.0)

[warning] `__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)